Resources/doc/index.rst

```diff
@@ -0,0 +1,45 @@
+Installation
+============
+
+Step 1: Download the Bundle
+---------------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+.. code-block:: bash
+
+    $ composer require symfony/monolog-bundle "~2.7"
```

Comment:
I'd prefer to not specify the version only `composer require symfony/monolog-bundle` is enough.